examples/abitest/client/config.h

```diff
@@ -23,21 +23,20 @@ node_configs {
   name: "frontend-config"
   wasm_config {
     module_bytes: "<filled in later>"
-    main_entrypoint: "frontend_oak_main"
   }
 }
```

Comment:
It occurred to me that one of the reasons why we have the application configuration as protobuf right now, is that we don't have a good way for the running node to pass this to the runtime.

If we had, say, a FIDL interface defining how to create new nodes, then we could define this schema there, with the various node types and their options, etc, and essentially drive all this from within the wasm node, which would create the FIDL objects and pass them to the runtime via either a dedicated call (e.g. `node_create`), or over standard channels (`channel_write`), after having obtained a reference to the runtime "node creation service" (whatever that may represent). It does not even need to be FIDL, it may be the same exact protobuf we currently use, and just pass that from the Wasm node to the runtime, instead of passing a string that is used to look up the actual config from this file.

I feel this would make the application more self-contained, and easier for developers to maintain and evolve over time. 

One caveat is that this would require adding a dependency on the IDL to the SDK used to support each language, e.g. it would not work well if Rust didn't have protobuf (or FIDL).

Something to think about for the future anyways, not saying it shsould belong to this PR.